src/web/event-notifications/event-notification-types/SlackNotificationForm.jsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import lodash from 'lodash';
+
+import { Input } from 'components/bootstrap';
+import FormsUtils from 'util/FormsUtils';
+import { Button, ControlLabel, FormGroup, HelpBlock } from 'components/graylog';
+import { ColorPickerPopover } from 'components/common';
+import ColorLabel from 'components/sidecars/common/ColorLabel';
+
+class SlackNotificationForm extends React.Component {
+    static propTypes = {
+      config: PropTypes.object.isRequired,
+      validation: PropTypes.object.isRequired,
```

Comment:
Can we make these [`shape`](https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes) props?

Reply:
I will look into shape props.